editor/src/clj/editor/game_object_common.clj

```diff
@@ -38,14 +38,12 @@
 
 (def component-transform-property-keys (set (keys scene/identity-transform-properties)))
 
-(defn- template-pb-map-raw [workspace resource-type]
+(defn template-pb-map [workspace resource-type]
   (let [template (workspace/template workspace resource-type)
         read-fn (:read-fn resource-type)]
     (with-open [reader (StringReader. template)]
       (read-fn reader))))
 
-(def template-pb-map (fn/memoize template-pb-map-raw))
```

Comment:
I removed `fn/memoize` as we agreed (because it's a lightweight operation rarely used)